studio/schemas/documents/resource.js

```diff
@@ -125,6 +125,27 @@ export default {
         ],
       },
     },
+    {
+      name: 'displayComponent',
+      type: 'string',
+      title: 'Display Component',
+      description:
+        'What component should be used in egghead-next to display this feature?',
+      // hidden: ({document}) => document.type !== 'feature',
```

Comment:
@Creeland hiding this field based on document type wasn't working in nested resources... not sure if this was a bug or something that I was doing wrong.